utils/handlers/graphql.go

```diff
@@ -175,16 +178,27 @@ func HandleGetCollections(adminMan *admin.Manager, schema *schema.Schema) http.H
 		dbType := vars["dbType"]
 		project := vars["project"]
 
-		collections, err := schema.GetCollectionName(ctx, project, dbType)
-		if err != nil {
+		switch utils.DBType(dbType) {
+		case utils.Mongo, utils.MySQL, utils.Postgres:
+			collections, err := crud.GetCollections(ctx, project, dbType)
```

Comment:
You don't need the switch case here

Reply:
so there won't be any other database than these 3

Reply:
The crud module takes care of that. In the future we could have more databases